agent/bench-scripts/test-bin/require-rpm

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+args="${@}"
+echo "${0} ${args}" >> ${_testlog}
```

Comment:
Actually, we don't want it to function nicely if `_testlog` is undefined, that would be mean the test environment is broken.

Reply:
That being the case, then the should should (have) validated `_testlog` explicitly so that it can report a properly descriptive error rather than letting the shell report a syntax error (e.g.) if it is undefined.